tests/test/test.js

```diff
@@ -48,22 +49,32 @@ function blenderRoundtripGltf(blenderVersion, gltfPath, outDirName, done, option
 }
 
 function validateGltf(gltfPath, done) {
-    const { exec } = require('child_process');
-    const cmd = `${gltfValidatorCommand} ${gltfPath}`;
-    var prc = exec(cmd, (error, stdout, stderr) => {
-        //if (stdout) process.stdout.write(stdout);
-        //if (stderr) process.stderr.write(stderr);
-
-        if (error) {
-            //console.error(`exec error: ${error}`);
-            done(error);
-            return;
-        }
-        //console.log(`stdout: ${stdout}`);
-        //console.log(`stderr: ${stderr}`);
-        done();
+    const asset = fs.readFileSync(gltfPath);
+    validator.validateBytes(new Uint8Array(asset), {
```

Comment:
I don't know Docker or CircleCI, but I can at least vouch for this correct usage of the npm-based glTF Validator.  Alexey (the author of the Validator) allows me to publish the npm package, and I use it in the VSCode plugin with code that looks functionally the same as this.  So this part of the PR is 👍